### PR TITLE
Refactor/related articles package

### PR DESCRIPTION
**Pull request by colincclark**
https://nidigitalsolutions.jira.com/browse/REPLAT-1770
https://github.com/newsuk/times-components/issues/617

- move related articles to it's own package
- separate stories in storybook
- fix some legacy layout issues

### CODE REVIEW

File: packages/related-articles/related-articles.stories.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { storiesOf } from "@storybook/react-native";
+import React from "react";
+import { ScrollView } from "react-native";
+import { decorateAction } from "@storybook/addon-actions";
+import { storybookReporter } from "@times-components/tealium";
+import RelatedArticles from "/";
+
+// Related articles
```
Comment (fourlastor) [inline packages/related-articles/related-articles.stories.js]: I think we can drop the comment now

File: packages/related-articles/related-articles.stories.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import { storiesOf } from "@storybook/react-native";
+import React from "react";
+import { ScrollView } from "react-native";
+import { decorateAction } from "@storybook/addon-actions";
+import { storybookReporter } from "@times-components/tealium";
+import RelatedArticles from "/";
```
Comment (fourlastor) [inline packages/related-articles/related-articles.stories.js]: the from bit feels a bit weird, should this be `./something.js`?

File: packages/related-articles/package.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  "peerDependencies": {
+    "react": ">=16",
+    "react-dom": ">=16",
+    "react-native": ">=0.50",
+    "react-native-web": ">=0.1"
+  },
+  "publishConfig": {
+    "access": "public"
+  }
+}
```
Comment (fourlastor) [inline packages/related-articles/package.json]: missing newline

File: packages/related-articles/related-articles.stories.js (hunk 1 above)
Comment (colincclark) [reply packages/related-articles/related-articles.stories.js]: done

File: packages/related-articles/related-articles.stories.js (hunk 2 above)
Comment (colincclark) [reply packages/related-articles/related-articles.stories.js]: done

File: packages/related-articles/package.json (hunk 3 above)
Comment (colincclark) [reply packages/related-articles/package.json]: done

Comment (colincclark) [reply packages/related-articles/package.json]: not sure why this is not showing on the PR - but it's there!!

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/7df6da7f35ad3557e1a699cad0784175efe6c179/index.html

**Pull request merged**